examples/update-nfts-metadata-of-non-fungible-token.js

```diff
@@ -118,19 +118,20 @@ async function main() {
         const newAccountId = accountCreateTxReceipt.accountId;
         console.log(`New account id: ${newAccountId.toString()}`);
 
-        const tokenAssociateTx = new TokenAssociateTransaction()
-            .setAccountId(newAccountId)
-            .setTokenIds([tokenId])
-            .freezeWith(client);
-
-        const tokenAssociateTxResponse = await (
-            await tokenAssociateTx.sign(operatorKey)
-        ).execute(client);
-        const tokenAssociateTxReceipt =
-            await tokenAssociateTxResponse.getReceipt(client);
-        console.log(
-            `Status of token associate transaction: ${tokenAssociateTxReceipt.status.toString()}`,
-        );
+        //Associating a token to an account is only required if the account does not have any automatic token association slots. The account we created in this example has 10
+        //const tokenAssociateTx = new TokenAssociateTransaction()
+            //.setAccountId(newAccountId)
+            //.setTokenIds([tokenId])
+            //.freezeWith(client);
+
+        //const tokenAssociateTxResponse = await (
+           // await tokenAssociateTx.sign(operatorKey)
+        //).execute(client);
+        //const tokenAssociateTxReceipt =
+            //await tokenAssociateTxResponse.getReceipt(client);
+        //console.log(
+            //`Status of token associate transaction: ${tokenAssociateTxReceipt.status.toString()}`,
+        //);
```

Comment:
Commented code should not be pushed. In my opinion we can get rid of both the comment and the commented code. The purpose of the example is showcasing HIP-657 and not token associations. Is there a reason not to remove these lines?